### DISCUSSION TITLE
好貴的課本

### DISCUSSION THREAD

**Discussion by eason8479** [1101-GS4719-C/Python-Community-GS4719-C#851] [閒聊八卦]
https://www.youtube.com/watch?v=sseSi0k3Ecg

大家應該上大學後都對教科書的價格都很有感
不過應該也貴不過影片中的課本吧?
(如果有的話...大哥/大姊，您好，小的見識短淺，如有冒犯，獻上萬分歉意)

至於原理到底是什麼，還是讓大家自己去影片裡看吧
非常非常簡而言之，就是兩隻機器人炒出來的

不知為何，寫程式時這種問題還蠻常見的，一不小心程式就炸了
就像下面這樣

```
i = 0
while (i < 100):
    i = i - 1
    print(i)
```

**Comment by williamyen0623** [comment]
記憶體在哭泣：太...大...了...不要再進來了QAQ

**Comment by evanchou1204** [comment]
文中的程式範例確實一不小心就會炸了，寫迴圈的時候真的要注意條件。

**Comment by ching020721** [comment]
寫迴圈真的要注意不要變無窮迴圈，不然記憶體真的會哭

**Comment by Evan0979** [comment]
滿明顯的bug吧,寫的時候稍微注意一下就好了

**Comment by CPCpyt** [comment]
又學到了，感謝分享，不過這小小一行看起來真的好可怕...

**Comment by StevenWanglolz** [comment]
還好這種菜雞無限迴圈錯誤我只有犯過一兩次哈哈哈，不過我到後來就不買教科書了，真的沒有必要....
